### DISCUSSION TITLE
How to use the timestamp_in_ms using windows command line to get it

### DISCUSSION THREAD

**Discussion by heemanshuGupta** [rabbitmq/rabbitmq-server#2662] [Other] [answered]
I am using the plugin rabbitmq_message_timestamp-3.8.0.ez and able to enable it successful. I can see the timestamp in millisecond in the message property headers: timestamp_In_ms:1606209373947 and same can be seen the attachment too.

But i do not know how to get the property/timestamp_in_ms using window command line by executing rabbitmqctl.bat list_queues or please suggest some alternative

i want to get the timestamp in milisecond to use it.

Please help in that issue.

**Comment by michaelklishin** [comment, marked as answer]
Message properties are only available to consumers and `rabbitmqctl` does not include one. We highly recommend against polling consumers, so perhaps you should re-consider basing your design on this property fetched by a polling consumer on the command line. Use a regular long-running consumer instead.
